tests/integration/questionnaire/test_questionnaire_progress_value_source_calculated_summary.py

```diff
@@ -0,0 +1,363 @@
+from tests.integration.integration_test_case import IntegrationTestCase
+
+
+class TestQuestionnaireProgressValueSource(IntegrationTestCase):
```

Comment:
Can you add some comments for these, these are hard to follow.

Reply:
Comments added all the way through